app-web/src/constants/designTokens.js

```diff
@@ -0,0 +1,59 @@
+export const STYLE_BASE = {
+  fontSize: 16,
+  spacing: 5,
+  borderWidth: 1,
+};
+
+// custom font size typing
+// CUSTOM_TYPE.sm => 16px;
+export const CUSTOM_TYPE = {
+  sm: `${STYLE_BASE}px`,
+  md: `${STYLE_BASE * 1.5}px`,
+  lg: `${STYLE_BASE * 2}px`,
+};
+
+export const MAX_SPACE_UNIT_MULTIPLE = 20;
+
+/**
+ * returns an object of spacing units for things like padding/margin
+ * @param {Number} maxSpacingMultiple the maximum multiple of the spacing factor
+ * @param {Number} spacingFactor
+ * @returns {Object} => {'1x': '8px', '2x': '16px'};
+ */
+const getSpacingUnits = (maxSpacingMultiple, spacingFactor) => {
+  const spacingUnits = {};
+  for (let i = 1; i <= maxSpacingMultiple; i++) {
+    spacingUnits[`${i}x`] = `${i * spacingFactor}px`;
```

Comment:
so does this object get generate everytime you call SPACING? would it be easier if it's a function that returns like `20px` instead?

Reply:
I think `SPACING` is evaluated only once